src/main/java/com/leetcode/problems/CalculateMoney.java

```diff
@@ -0,0 +1,16 @@
+package com.leetcode.problems;
+
+
+public class CalculateMoney {
+  public static void main(String[] args) {
+    System.out.println(totalMoney(4));
+  }
+
+  public static int totalMoney(int n) {
+    int completWeeks = n / 7;
+    int restOfDays = (n % 7);
+    int completeWeeksSum = (28 * completWeeks) + 7 * (completWeeks)*(completWeeks-1)/2;
```

Comment:
Is this really formatted?

Check the spacing here "(28 * completWeeks)" and here "(completWeeks-1)" they do not match. Please run the formatter over this file.

Also add the problem URL. This solution is what I also had in mind, did you test it ok LeetCode?

Reply:
Now isproperly tested and works fine on leetcode.